build_all.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
How important is it to have a `build_all` script with all versions and requiring this `gvm` thing? 

To ensure that the library is properly building against every supported version, I'd setup a Travis build (free for open source).

From then on, if someone wants to use different Go version locally - they would be welcome to run `gvm` and then build and tests.